plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/CachingDirectoryLister.java

```diff
@@ -246,8 +254,9 @@ public long getRequestCount()
     @Override
     public boolean isCached(Location location)
```

Comment:
This is the method in a hot path (via `BackgroundHiveSplitLoader`) that should be able to provide both the table name and location to do a point lookup instead of traversing the cache entries.

Reply:
oh cool, i wasn't sure if i could change these API signatures, ill go ahead and take a look and see what i can do. Thanks for the headsup